server.js

```diff
@@ -145,9 +145,9 @@ function initialise () {
 
   app.use(flash())
   initialiseTemplateEngine(app)
-  initialiseRoutes(app)
   initialisePublic(app)
   initialiseErrorHandling(app)
+  initialiseRoutes(app)
```

Comment:
If it is imperative that this comes last it would be good to add a comment here to protect against future mishaps, as you've done for app/router.js.